Userland/Applications/KeyboardSettings/KeyboardSettingsWidget.cpp

```diff
@@ -228,7 +228,13 @@ KeyboardSettingsWidget::KeyboardSettingsWidget()
     m_selected_keymaps_listview->on_selection_change = [&]() {
         auto& selection = m_selected_keymaps_listview->selection();
         m_remove_keymap_button->set_enabled(!selection.is_empty() && keymaps_list_model.keymaps().size() > 1);
-        m_activate_keymap_button->set_enabled(!selection.is_empty());
+        if (selection.is_empty()) {
+            m_activate_keymap_button->set_enabled(!selection.is_empty());
```

Comment:
We can simplify `!selection.is_empty()` to `false` here. Means you don't have to look at the if again.

Reply:
The selection argument in this callback can be empty when we click the empty area of the list view though (The one highlighted in the screenshot)
![kb_selection_highlight](https://user-images.githubusercontent.com/20781/206840877-7b2f270d-e73b-4352-8065-d6cb10d6d77e.png)

Reply:
Oh I mean in the if branch, since we checked that the selection is empty in the if itself we know that `!selection.is_empty()` must be false.